lib/src/wizard/Wizard.tsx

```diff
@@ -1,9 +1,8 @@
-// @ts-nocheck
 import React, { useState } from "react";
 import styled, { ThemeProvider } from "styled-components";
 import { spaces } from "../common/variables.js";
 import useTheme from "../useTheme";
-import WizardPropsType from "./types";
+import WizardPropsType, { Margin, Mode, Space } from "./types";
 
 const icons = {
```

Comment:
Maybe export the icons from a different file?

Reply:
Since there are only two icons I think it is unnecessary to extract them to another file. I only take them out when they are a lot and increase the component's code considerably.